source/manuals/programming-languages/docker.html.md.erb

```diff
@@ -31,7 +38,10 @@ where:
 As you cannot rely on the tag pointing to the exact same image over time, you
 should instead use a digest, which identifies the image by a hash of its
 contents. This makes sure that you are always referencing the image that you
-expect.
+expect.  Add a comment above the FROM line with the intended tag and version.
+Though dependabot will not use this, please update it when the digest changes
+so that it is possible to understand the magnitude of the change by knowing
+the significance of the version change.
 
 For example:
```

Comment:
The example should include the comment, really

Reply:
tbh actually the new 41-45 feel superfluous given the new guidance below about including the tag and the comment.

Why not just remove all the examples except the example that shows how we _want_ this to be done, which is the new one at line 78?

Reply:
I just did it, hope that's ok